frb/tests/test_frbsurveys.py

```diff
@@ -97,3 +99,29 @@ def test_first():
     assert isinstance(first_tbl, Table)
     assert len(first_tbl) == 1
 
+def test_panstarrs():
```

Comment:
For some reason this is failing Travis. 
Probably it won't execute the HTML pull.
Not sure.  But please modify until it does pass
(even if you need to kludge it).

Thanks.

Reply:
It was a bug in my code. It should pass now.